k8s_handle/k8s/diff.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import logging
+import copy
+from difflib import ndiff
+from datetime import datetime
+from functools import reduce
+import operator
+import yaml
+from .adapters import Adapter
+from k8s_handle.templating import get_template_contexts
+log = logging.getLogger(__name__)
+
+IGNORE_FIELDS = [
```

Comment:
What about `metadata.managedFields` (was added in 1.18)? It looks like it should be ignored as well.

Example:
```
  apiVersion: apiextensions.k8s.io/v1beta1
  kind: CustomResourceDefinition
  metadata:
-   managedFields:
-   - apiVersion: apiextensions.k8s.io/v1beta1
-     manager: Swagger-Codegen
-     operation: Update
-     time: '2020-10-19 07:14:08+00:00'
-   - apiVersion: apiextensions.k8s.io/v1
-     manager: kube-apiserver
-     operation: Update
-     time: '2020-10-19 07:14:13+00:00'
    name: tests.test.example.com
```

Reply:
Added, please check.